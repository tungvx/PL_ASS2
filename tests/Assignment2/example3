A=test
B=more complex
C=with multiple lines
%
this is a B A
The file contains C.

and even an empty line!
