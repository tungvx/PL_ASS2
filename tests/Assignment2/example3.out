this is a more complex test
The file contains with multiple lines.

and even an empty line!
